pages/agent/v3/windows.md.erb

```diff
@@ -4,7 +4,16 @@ The Buildkite Agent can be installed on both 64bit and 32bit editions of Windows
 
 <%= toc %>
 
-## Installation
+## Installing with Powershell
```

Comment:
We should also update the docs here: https://github.com/buildkite/frontend/blob/master/app/components/docs/07-windows.mdx

Reply:
Will do!